nodewatcher/nodewatcher.py

```diff
@@ -11,23 +11,18 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
-__author__ = 'dougalb'
-
-from datetime import datetime
-import urllib2
+import ConfigParser
+import json
+import logging
 import os
-import time
 import sys
 import tempfile
-import logging
+import time
+import urllib2
```

Comment:
`urllib2` is just a python 2 package. Breaks interoperability. See https://stackoverflow.com/questions/2792650/import-error-no-module-name-urllib2

Reply:
I just re-ordered the imports, `urllib2` was already there. 
I think this can be addressed in another patch, if required.